sdk/aqueduct/integrations/dynamic_k8s_integration.py

```diff
@@ -64,8 +64,13 @@ def delete(self) -> None:
             "Cluster is currently in %s status. It could take 6-8 minutes for the cluster to be terminated..."
             % status.value
         )
-        globals.__GLOBAL_API_CLIENT__.edit_engine(
-            action="delete", integration_id=str(self._metadata.id)
+
+        action = "delete"
+        if force:
+            action = "force-delete"
```

Comment:
We should make `create`, `delete` and `force-delete` an enum so it's clear what possible "actions" there are.